app/src/controllers/testItem/sagas.js

```diff
@@ -190,10 +192,22 @@ function* fetchTestItems({ payload = {} }) {
 
   const isFilterNotReserved = !FILTER_TITLES[filterId];
   if ((isTestItemsList || isFilterNotReserved) && !activeFilter) {
-    const filter = yield call(fetch, URLS.filter(project, filterId));
-
-    if (filter) {
-      yield put(showFilterOnLaunchesAction(filter));
+    try {
+      const filter = yield call(fetch, URLS.filter(project, filterId));
+      if (filter) {
+        yield put(showFilterOnLaunchesAction(filter));
+      }
+    } catch {
+      const testItemIds = yield select(testItemIdsSelector);
+      const link = {
+        type: TEST_ITEM_PAGE,
+        payload: {
+          filterId: ALL,
+          projectId: project,
+          testItemIds,
+        },
```

Comment:
For correct redirection to the required page, please provide the `query` property to receive all necessary parameters from URL:
```
meta: {
   query: pageQuery,
},
```